mlir/include/mlir/Dialect/Rock/Tuning/GridwiseGemmParams.h

```diff
@@ -272,7 +270,8 @@ class PopulateParamsXDL : public BasePopulateParams<InitParamsXDL> {
 
   LogicalResult isValidBlockwiseGemmXDLOPS(const InitParamsXDL &param,
                                            Type dataTypeA, Type dataTypeB,
-                                           StringRef arch, uint32_t blockSize);
+                                           StringRef arch, int64_t waveSize,
```

Comment:
nit : given we look up waveSize using arch, the expansion of func signature is to save that lookup ?

Reply:
Yeah, I agree, it's probably better to look up again